### ISSUE TITLE
Modifications that might affect accessibility

### ISSUE THREAD

**Issue by paaljoachim** [wplemon/gridd#90]
Hey Ari

The customer who I am remaking this web site for: https://celias.no/ Going from Avada to Gridd had a few requests. To remove the following that shows up on hover around all the menu links.

![image001](https://user-images.githubusercontent.com/5323259/63415502-20264800-c3fe-11e9-8047-44a9f52773a9.png)

I added the following CSS.
```
/* Remove outline: thin dotted square when hovering over menu items. */
.gridd-site-wrapper a:active, .gridd-site-wrapper a:focus, .gridd-site-wrapper a:hover {
    color: var(--gridd-links-hover-color,#541cfc);
    outline: none !important; 
}
```

The customer also asked me to remove the blue focus outline around objects when they are in focus. I adjusted with the following.
```
/* Changed focus border color around title of accordion when hovering over content inside of it. 
 * outline: -webkit-focus-ring-color auto 5px; */
:focus {
	outline: #76a8b0 auto 1px !important;
}
```

I also changed the underline and menu link color that shows up on hover. Added the following.
```
/* Hover color - links. Underline color - links */
.gridd-tp-nav_1 .gridd-navigation li.current-menu-ancestor>a, .gridd-tp-nav_1 .gridd-navigation li.current-menu-ancestor>a:visited, .gridd-tp-nav_1 .gridd-navigation li.current-menu-item>a, .gridd-tp-nav_1 .gridd-navigation li.current-menu-item>a:visited, .gridd-tp-nav_1 .gridd-navigation li.current-menu-parent>a, .gridd-tp-nav_1 .gridd-navigation li.current-menu-parent>a:visited, .gridd-tp-nav_1 .gridd-navigation li.current-page-ancestor>a, .gridd-tp-nav_1 .gridd-navigation li.current-page-ancestor>a:visited, .gridd-tp-nav_1 .gridd-navigation li.current-page-item>a, .gridd-tp-nav_1 .gridd-navigation li.current-page-item>a:visited, .gridd-tp-nav_1 .gridd-navigation li.current-page-parent>a, .gridd-tp-nav_1 .gridd-navigation li.current-page-parent>a:visited, .gridd-tp-nav_1 .gridd-navigation li.current_page_ancestor>a, .gridd-tp-nav_1 .gridd-navigation li.current_page_ancestor>a:visited, .gridd-tp-nav_1 .gridd-navigation li.current_page_item>a, .gridd-tp-nav_1 .gridd-navigation li.current_page_item>a:visited, .gridd-tp-nav_1 .gridd-navigation li.current_page_parent>a, .gridd-tp-nav_1 .gridd-navigation li.current_page_parent>a:visited {
    color: #76a8b0 !important;
    border-bottom: 2px solid;
}
```

The reason why I mention the above is that these are likely things a person might ask about how to adjust. It would be helpful to add a mention in the docs on how to adjust the above so that accessibility is still in nice working order, and that the customer is also satisfied. Let me know if I should adjust it differently. Thanks.

**Comment by aristath** [comment]
Yes, these changes will effect accessibility negatively.
`hover` and `focus` states should be different.
Removing the `outline` is acceptable only if it gets replaced by some other method of differentiating them - other than color.
You could try something like `box-shadow:inset 0px -10px` for example and experiment. But removing the `outline` without replacing it with something else is one of the most common mistakes.

Further reading:
* http://www.outlinenone.com/
* https://www.w3.org/TR/2008/NOTE-WCAG20-TECHS-20081211/F78
* https://webaim.org/blog/plague-of-outline-0/
* https://a11yproject.com/posts/never-remove-css-outlines/

**Comment by aristath** [comment]
@paaljoachim I added a new option in the nav menus, if you use the dev branch it's the last option in the navigation grid-part options 👍

**Comment by paaljoachim** [comment]
Do you mean the Hover/Focus styles?

**Comment by aristath** [comment]
yes
